Added active jobs dashboard example

ThePrez opened the pull request:
Requirements include jQuery and tablesorter, will add those in README.md in next commit

ThePrez committed:
Added active jobs dashboard example

ThePrez committed:
Create README.md

ThePrez committed:
Add screen shot

ThePrez committed:
Update README.md

jordiwes commented on non-wheel/active-jobs-dashboard/root.tpl:
```diff
@@ -0,0 +1,64 @@
+<html>
+<head><title>Active Job Dashboard</title>
+<script type="text/javascript" src="jquery-latest.js"></script>
```
I would suggest using the CDN here: <script
  src="https://code.jquery.com/jquery-3.2.0.min.js"
  integrity="sha256-JAW99MJVpJBGcbzEuXk4Az05s/XyDdBomFqNlM3ic+I="
  crossorigin="anonymous"></script>

ThePrez replied:
I have not seen scripts included with the "integrity" and "crossorigin" attributes before, but trust your more-able insight that this is secure. :)

jordiwes commented on non-wheel/active-jobs-dashboard/root.tpl:
```diff
@@ -0,0 +1,64 @@
+<html>
+<head><title>Active Job Dashboard</title>
+<script type="text/javascript" src="jquery-latest.js"></script>
+<script type="text/javascript" src="jquery.tablesorter.min.js"></script>
```
I'm thinking it might be easier for the user if we include jquery.tablesorter.min.js in this repo, and style.css too :)

ThePrez replied:
Yes indeed, this will make it much more usable. Should be able to "git clone" and go!

ThePrez committed:
Updates made in response to PR comments

Included tablesorter files, changed jquery include to pull from public
jquery source, simplified README,

ThePrez committed:
Removed jquery and tablesorter dependencies from README

ThePrez commented:
Thanks for the comments! Submitted new commits for your review and consideration

jordiwes commented:
Great! Got it all working on our box once we updated to python 3.4.4
